src/app/page.tsx

```diff
@@ -1,4 +1,4 @@
-import Image from "next/image";
+// import Image from "next/image";
 // import styles from "./page.module.css";
```

Comment:
Any reason to retain these comments?

Reply:
Not really. I was just testing the Linter